docs/app/pages/components/components-sections/hierarchy-bar/hierarchy-bar/hierarchy-bar.component.html

```diff
@@ -41,7 +48,9 @@
         If specified, this node will be selected when the component is initialized.
     </tr>
 
-    <tr uxd-api-property name="mode" type="'standard' | 'collapsed'" defaultValue="standard">
+    <tr uxd-api-property name="mode"
+        type="'standard' | 'collapsed' | 'alternativeClickBehavior' | 'readOnly'"
```

Comment:
I'm also wondering should we simplify `alternativeClickBehavior`, I don't think `Behavior` is required on the name as they are all essentially `behaviors`. From what I can tell this is an alternate of the `standard` mode, so perhaps `standard-alt`? Any thoughts @donna-marijne?

Reply:
Yes I think it could do with a better name. The emphasis is on the availability of the dropdowns so maybe it could be "dropdown" mode. (Just trying to think of something descriptive.)